### PR TITLE
Medical form labels questions - update

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@erikh360 Updated the "includes/receipt.html" template as it's no longer an include template but standalone.

File: higher_health/views.py [hunk 1]
```diff
@@ -28,14 +27,14 @@ def healthcheck_receipt(request):
     if "saved_data" in request.session:
         data = request.session.get("saved_data")
         del request.session["saved_data"]
-        return render(request, "includes/receipt.html", data)
+        return render(request, "healthcheck_receipt.html", data)
```
Comment (erikh360) [inline higher_health/views.py:30]: Why are we changing this and not just rendering the receipt page?

**Review by erikh360 — COMMENTED** [review]
I'm a little confused by the receipt page being rendered inside the question page

File: higher_health/views.py (hunk 1 above)
Comment (Mitso) [reply higher_health/views.py:30]: This should be a standalone page?

Comment (erikh360) [reply higher_health/views.py:30]: sorry I misread that completely, all good

**Pull request merged**